### DISCUSSION TITLE
Odd issue

### DISCUSSION THREAD

**Discussion by stepdadterry** [isid0re/SoloLeveling#220] [Q&A]
The exe keeps crashing soon as i make a game in a1 with a new character. Tried downloading a fresh kolbot from most recent github, no luck there. Doesnt give me any red/orange text errors before it goes down. Just about 3 seconds after i get into the game the whole client crashes and starts the whole restart back up

Ive also tried multiple different classes with the same result.

Anyone ran into this problem or have any idea how to fix it?

**Comment by isid0re** [comment]
Someone reported this on discord yesterday for the assassin.

The paladin loaded fine. During a brief testing to pinpoint. 

Can you post what classes crashed for you?

Thanks.

**Reply by isid0re** [reply]
Did a push that fixes a bunch of crashes. May resolve your issue. Please update and let me know
